packages/react/src/components/cta/Button/index.tsx

```diff
@@ -18,7 +18,7 @@ interface BaseProps {
   disabled?: boolean;
 }
 
-export interface ButtonProps extends BaseProps {
+export interface ButtonProps extends BaseProps, SharedStyleProps {
```

Comment:
If i let BaseProps, i have a conflict between the color from sharedStyleProps, and some color inherited from an unknown place
I will do it another way to keep BaseProps

Reply:
ButtonProps is used for the Button component, if we don't put it in, we can't use the styled-system props
I may put it for the two interface

Reply:
> ButtonProps is used for the Button component, if we don't put it in, we can't use the styled-system props from sharedStyle

Hmm this is weird since ButtonProps extends BaseProps?